### PR TITLE
fixing create profile

### PR DESCRIPTION
**Pull request by benduongcmc**

### CODE REVIEW

**Comment by mrhillman** [comment]
Can we revert all package lock and yarn lock changes here too. No need to change them here

File: android/src/main/java/com/waivpaykartasdk/AddToWallet.java [hunk 1]
```diff
@@ -31,119 +33,141 @@
...
 public class AddToWallet {
-    public static final String HOST_STAGING = "https://webstores-staging.herokuapp.com/";
-    public static final String HOST_PRODUCTION = "https://webstore.egivv.com/";
+  public static final String HOST_STAGING = "https://webstores-staging.herokuapp.com/";
+  public static final String HOST_PRODUCTION = "https://webstore.egivv.com/";
+  private static final int REQUEST_CREATE_WALLET = 4;
+
+  public AddToWallet() {
+
+  }
```
Comment (mrhillman) [inline android/src/main/java/com/waivpaykartasdk/AddToWallet.java:42]: Is this right? It's an empty block

File: android/src/main/java/com/waivpaykartasdk/AddToWallet.java [hunk 2]
```diff
@@ -31,119 +33,141 @@
...
                             }
-                        });
-    }
-
+                            promise.resolve(false);
+                          }
+                        }
+                      });
+                } else {
+                  showSimpleDialog("Error", "Error getting wallet id", activity);
```
Comment (mrhillman) [inline android/src/main/java/com/waivpaykartasdk/AddToWallet.java:166]: Can be removed

File: android/src/main/java/com/waivpaykartasdk/AddToWallet.java [hunk 3]
```diff
@@ -31,119 +33,141 @@
...
+        promise.resolve(true);
+      } else {
+        Log.w("isReadyToPay failed", completedTask.getException());
+        promise.resolve(false);
+      }
+    });
+
+  }
+
+  private void showSimpleDialog(String title, String message, Activity activity) {
```
Comment (mrhillman) [inline android/src/main/java/com/waivpaykartasdk/AddToWallet.java:130]: Can remove this too

**Review by mrhillman — CHANGES_REQUESTED** [review]

**Comment by mrhillman** [comment]
And as above comment, let's revert package lock and yarn lock changes, keep it clean

File: android/src/main/java/com/waivpaykartasdk/AddToWallet.java (hunk 1 above)
Comment (benduongcmc) [reply android/src/main/java/com/waivpaykartasdk/AddToWallet.java:42]: yes, this is the constructor in Java; it had this one before. It's like the initial declaration.

File: src/ApiCall.ts [hunk 4]
```diff
@@ -931,7 +939,7 @@ export async function generateBarcode(
 }
 
 // verify phone number
-export async function verifyPhoneNumber(phoneNumber: string) {
+export async function verifyPhoneNumber(phoneNumber: string, userId?: number) {
```
Comment (mrhillman) [inline src/ApiCall.ts:942]: This is not right, this is not needed here. Only required for `sendTwoFactor`

**Review by mrhillman — COMMENTED** [review]
Cool, just about there. Just the one comment. Can we also revert package lock changes so there's no change here. Probably not a big deal, but prefer to leave untouched

**Review by mrhillman — APPROVED** [review]

**Pull request merged**